backend/package.json

```diff
@@ -1,15 +1,20 @@
 {
   "dependencies": {
+    "@prisma/client": "^5.17.0",
+    "cors": "^2.8.5",
     "express": "^4.19.2"
   },
   "name": "backend",
   "version": "1.0.0",
   "description": "server side of skillswap",
   "main": "server.js",
-  "devDependencies": {},
+  "devDependencies": {
+    "nodemon": "^3.1.4"
+  },
   "scripts": {
```

Comment:
Is this what you mean by script? So "start" : "node server.js" or "start" : "nodemon server.js"